mne/viz/utils.py

```diff
@@ -385,19 +387,24 @@ def _draw_proj_checkbox(event, params, draw_current_state=True):
 
     labels = [p['desc'] for p in projs]
     actives = ([p['active'] for p in projs] if draw_current_state else
-               [True] * len(params['projs']))
+               [params['apply_proj']] * len(projs))
 
     width = max([len(p['desc']) for p in projs]) / 6.0 + 0.5
```

Comment:
While you're at it, please add a minimum width to the figure. If the proj names are short it cuts off the title. I tested and seems like 4 is a good default size so change this line to 
``width = max([max([len(p['desc']) for p in projs]) / 6.0 + 0.5, 4.])``